src/components/navbar/navbar-item.js

```diff
@@ -0,0 +1,77 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+
+
+const NavbarItem = forwardRef(({
+  id,
+  classes,
+  label,
+  icon,
+  onClick,
+  isSelected,
+  ...rest
+}, ref) => {
+  const NavbarItemClasses = Object.freeze({
+    itemContainer: `navbar-item__main-container ${classes.itemContainer} cursor-pointer mx-6`,
+    itemContainerActive: `navbar-item__main-container--active ${classes.itemContainerActive} py-2 px-4 mx-2 rounded-full bg-black`,
+    contentContainer: `navbar-item__content-container ${classes.contentContainer} flex flex-row items-center`,
+    icon: `navbar-item__icon ${classes.icon} pr-2.5`,
+    iconActive: `navbar-item__icon ${classes.iconActive} pr-2.5 text-white`,
+    text: `navbar-item__text ${classes.text} text-black`,
+    textActive: `navbar-item__text ${classes.textActive} text-white`,
```

Comment:
looks good lets add active for active classes so user can target then individually. navbar-item__icon-active